S3/English/ASS_31.ass

```diff
@@ -0,0 +1,108 @@
+﻿[Script Info]
+; Script generated by Aegisub 3.2.2
+; http://www.aegisub.org/
+ScriptType: v4.00+
+WrapStyle: 0
+ScaledBorderAndShadow: yes
+YCbCr Matrix: TV.601
+PlayResX: 1920
+PlayResY: 1080
+
+[Aegisub Project Garbage]
+Audio File: 万圣街_1920x1080_万圣街_31.mp4
+Video File: 万圣街_1920x1080_万圣街_31.mp4
+Video AR Mode: 4
+Video AR Value: 1.777778
+Video Zoom Percent: 0.375000
+Scroll Position: 57
+Active Line: 69
+Video Position: 5295
+
+[V4+ Styles]
+Format: Name, Fontname, Fontsize, PrimaryColour, SecondaryColour, OutlineColour, BackColour, Bold, Italic, Underline, StrikeOut, ScaleX, ScaleY, Spacing, Angle, BorderStyle, Outline, Shadow, Alignment, MarginL, MarginR, MarginV, Encoding
+Style: Default,Arial,45,&H00FFFFFF,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,100,0,0,1,2,2,2,18,0,20,1
+Style: Title,Arial,75,&H006CC2E9,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,100,0,0,1,2,2,2,10,10,280,1
+
+[Events]
+Format: Layer, Start, End, Style, Name, MarginL, MarginR, MarginV, Effect, Text
+Dialogue: 0,0:01:02.51,0:01:05.56,Title,,0,0,0,, {\pos(970,920)} {\b1\an5\pos(970,880)\fs120\frz4.5\t(0,100,1.5\fs45)}Episode 31:  {\c&H7990E7&}Workers' life{\b0}
+Dialogue: 0,0:01:06.84,0:01:08.00,Default,,0,0,0,,5 o'clock in the morning
+Dialogue: 0,0:01:08.80,0:01:12.30,Default,,0,0,0,,While Neil is still in a sweet sleep
+Dialogue: 0,0:01:12.98,0:01:15.99,Default,,0,0,0,,Ira finished his stream for today
+Dialogue: 0,0:01:16.45,0:01:18.75,Default,,0,0,0,,And Abu just get up
+Dialogue: 0,0:01:19.28,0:01:21.90,Default,,0,0,0,,Lynn is busy preparing breakfast
+Dialogue: 0,0:01:22.10,0:01:22.68,Default,,0,0,0,,Nick
+Dialogue: 0,0:01:22.87,0:01:26.40,Default,,0,0,0,,Luis is practicing as usual
+Dialogue: 0,0:01:22.88,0:01:24.36,Default,,0,0,0,,{\pos(970,200)\i1}A Chinese tongue twister that idk how to tl{\i0}
```

Comment:
maybe you can place an english tongue twister and specify it's not the actual TL in the description?

Reply:
^^ or you could always just say "* A Chinese tongue-twister for practising standard Mandarin." without translating it. It's not really written to be translatable for any meaningful purpose anyway